### DISCUSSION TITLE
Switching from mycroft-core help.

### DISCUSSION THREAD

**Discussion by XsqrdLabsAdmin** [OpenVoiceOS/ovos-buildroot#138] [Q&A] [answered]
Hello. I work for a company that was using mycroft as a base for it's AI voice assistant and recently after consulting with an ex mycroft employee we decided to make the switch to ovos. There is a problem though, the assistant is no longer connecting to any server. We, switched the config file, installed ovos-core and still nothing. If anyone could provide some assistance that would be greatly appreciated.

Specs: Raspberry pi 3B running picroft on a 16GB micro-SD card.

**Comment by JarbasAl** [comment, marked as answer]
what do you mean by "connecting to any server", the home-mycroft.ai portal? by default ovos-core does not use any backend server, you can enable it if you want 

https://openvoiceos.github.io/community-docs/backend

**Reply by XsqrdLabsAdmin** [reply]
it says, can not connect to mycroft's servers. I will try the suggested solution.

**Reply by XsqrdLabsAdmin** [reply]
Back up and running. Thank you for the assistance.

**Comment by ChanceNCounter** [comment]
I’d encourage you to join our support channel on Matrix:

https://matrix.to/#/%23openvoiceos-support%3Amatrix.org

In response to Mycroft winding down operations, we are scaling up, becoming a nonprofit, and adding helpers to our organization. You might have stumbled aboard a few weeks too early to experience a smooth onboarding, but supporting downstream businesses and projects is the whole point!
